### ISSUE TITLE
Message response failed when replying a large json response 

### ISSUE THREAD

**Issue by chaminda** [ballerinalang/ballerina#2064]
Following exception is noticed when a large response(json payload) is replying.

```
error in ballerina program: error occurred during writing the message to the output stream: null
	 at getIssueFromID(ValidateConnector.bal:197)
	 at testService(ValidateConnector.bal:183)

```

`{"root":{"expand":"renderedFields,names,schema,operations,editmeta,changelog,versionedRepresentations","id":"87475","self":"https://wso2.org/jira/rest/api/2/issue/87475","key":"IDENTITY-5751","fields":{"issuetype":{"self":"https://wso2.org/jira/rest/api/2/issuetype/4","id":"4","description":"An improvement or enhancement to an existing feature or task.","iconUrl":"https://wso2.org/jira/secure/viewavatar?size=xsmall&avatarId=11630&avatarType=issuetype","name":"Improvement","subtask":false,"avatarId":11630},"timespent":null,"customfield_10075":{"self":"https://wso2.org/jira/rest/api/2/customFieldOption/10039","value":"Major","id":"10039"},"project":{"self":"https://wso2.org/jira/rest/api/2/project/10041","id":"10041","key":"IDENTITY","name":"WSO2 Identity Server","avatarUrls":{"48x48":"https://wso2.org/jira/secure/projectavatar?pid=10041&avatarId=10011","24x24":"https://wso2.org/jira/secure/projectavatar?size=small&pid=10041&avatarId=10011","16x16":"https://wso2.org/jira/secure/projectavatar?size=xsmall&pid=10041&avatarId=10011","32x32":"https://wso2.org/jira/secure/projectavatar?size=medium&pid=10041&avatarId=10011"}},"fixVersions":[],"aggregatetimespent":null,"resolution":null,"customfield_10113":null,"resolutiondate":null,"workratio":-1,"lastViewed":null,"watches":{"self":"https://wso2.org/jira/rest/api/2/issue/IDENTITY-5751/watchers","watchCount":1,"isWatching":false},"created":"2017-02-14T10:21:02.000+0530","customfield_10020":null,"customfield_10220":null,"priority":{"self":"https://wso2.org/jira/rest/api/2/priority/1","iconUrl":"https://wso2.org/jira/images/icons/priorities/blocker.svg","name":"Highest","id":"1"},"customfield_10420":null,"labels":[],"timeestimate":null,"aggregatetimeoriginalestimate":null,"versions":[{"self":"https://wso2.org/jira/rest/api/2/version/12557","id":"12557","description":"6.0.0-M1","name":"6.0.0-M1","archived":false,"released":true,"releaseDate":"2017-01-28"}],"issuelinks":[{"id":"15490","self":"https://wso2.org/jira/rest/api/2/issueLink/15490","type":{"id":"10003","name":"Cloners","inward":"is cloned as","outward":"is a clone of","self":"https://wso2.org/jira/rest/api/2/issueLinkType/10003"},"outwardIssue":{"id":"87474","key":"IDENTITY-5750","self":"https://wso2.org/jira/rest/api/2/issue/87474","fields":{"summary":"Add support for sorting for user core and expose the same from product (SCIM) API","status":{"self":"https://wso2.org/jira/rest/api/2/status/1","description":"The issue is open and ready for the assignee to start work on it.","iconUrl":"https://wso2.org/jira/images/icons/statuses/open.png","name":"Open","id":"1","statusCategory":{"self":"https://wso2.org/jira/rest/api/2/statuscategory/2","id":2,"key":"new","colorName":"blue-gray","name":"To Do"}},"priority":{"self":"https://wso2.org/jira/rest/api/2/priority/1","iconUrl":"https://wso2.org/jira/images/icons/priorities/blocker.svg","name":"Highest","id":"1"},"issuetype":{"self":"https://wso2.org/jira/rest/api/2/issuetype/4","id":"4","description":"An improvement or enhancement to an existing feature or task.","iconUrl":"https://wso2.org/jira/secure/viewavatar?size=xsmall&avatarId=11630&avatarType=issuetype","name":"Improvement","subtask":false,"avatarId":11630}}}}],"assignee":{"self":"https://wso2.org/jira/rest/api/2/user?username=darshana%40wso2.com","name":"darshana@wso2.com","key":"darshana@wso2.com","emailAddress":"darshana at wso2 dot com","avatarUrls":{"48x48":"https://wso2.org/jira/secure/useravatar?avatarId=10133","24x24":"https://wso2.org/jira/secure/useravatar?size=small&avatarId=10133","16x16":"https://wso2.org/jira/secure/useravatar?size=xsmall&avatarId=10133","32x32":"https://wso2.org/jira/secure/useravatar?size=medium&avatarId=10133"},"displayName":"Darshana Gunawardana","active":true,"timeZone":"Asia/Colombo"},"updated":"2017-02-14T10:27:15.000+0530","status":{"self":"https://wso2.org/jira/rest/api/2/status/1","description":"The issue is open and ready for the assignee to start work on it.","iconUrl":"https://wso2.org/jira/images/icons/statuses/open.png","name":"Open","id":"1","statusCategory":{"self":"https://wso2.org/jira/rest/api/2/statuscategory/2","id":2,"key":"new","colorName":"blue-gray","name":"To Do"}},"components":[{"self":"https://wso2.org/jira/rest/api/2/component/11031","id":"11031","name":"user-core","description":"Carbon kernel APIs for user management and implementations for JDBC, LDAP and AD user store managers"}],"customfield_10090":{"self":"https://wso2.org/jira/rest/api/2/customFieldOption/10061","value":"Yes","id":"10061"},"timeoriginalestimate":null,"description":"One should be able to composite query like follows,\r\n\r\nList,\r\n* first 100 employees (pagination)\r\n* \"from sales department\" and \"who working from Sri Lanka\" (filtering)\r\n* in order of the date they joined to the company (sorting)\r\n\r\nThen it should be able to query the second set of employees (from 101 to 200) and so forth.\r\n","timetracking":{},"attachment":[],"aggregatetimeestimate":null,"summary":"Add support for filtering\\sorting\\pagination together for user core and expose the same from product (SCIM) API","creator":{"self":"https://wso2.org/jira/rest/api/2/user?username=darshana%40wso2.com","name":"darshana@wso2.com","key":"darshana@wso2.com","emailAddress":"darshana at wso2 dot com","avatarUrls":{"48x48":"https://wso2.org/jira/secure/useravatar?avatarId=10133","24x24":"https://wso2.org/jira/secure/useravatar?size=small&avatarId=10133","16x16":"https://wso2.org/jira/secure/useravatar?size=xsmall&avatarId=10133","32x32":"https://wso2.org/jira/secure/useravatar?size=medium&avatarId=10133"},"displayName":"Darshana Gunawardana","active":true,"timeZone":"Asia/Colombo"},"customfield_10080":null,"subtasks":[],"customfield_10040":{"self":"https://wso2.org/jira/rest/api/2/customFieldOption/10012","value":"Moderate","id":"10012"},"customfield_10084":null,"reporter":{"self":"https://wso2.org/jira/rest/api/2/user?username=darshana%40wso2.com","name":"darshana@wso2.com","key":"darshana@wso2.com","emailAddress":"darshana at wso2 dot com","avatarUrls":{"48x48":"https://wso2.org/jira/secure/useravatar?avatarId=10133","24x24":"https://wso2.org/jira/secure/useravatar?size=small&avatarId=10133","16x16":"https://wso2.org/jira/secure/useravatar?size=xsmall&avatarId=10133","32x32":"https://wso2.org/jira/secure/useravatar?size=medium&avatarId=10133"},"displayName":"Darshana Gunawardana","active":true,"timeZone":"Asia/Colombo"},"customfield_10120":null,"customfield_10121":null,"aggregateprogress":{"progress":0,"total":0},"customfield_10122":"9223372036854775807","customfield_10320":"0|i0cx87:","environment":null,"duedate":null,"progress":{"progress":0,"total":0},"comment":{"comments":[],"maxResults":0,"total":0,"startAt":0},"votes":{"self":"https://wso2.org/jira/rest/api/2/issue/IDENTITY-5751/votes","votes":0,"hasVoted":false},"worklog":{"startAt":0,"maxResults":20,"total":0,"worklogs":[]}}}}`

**Comment by chanakaudaya** [comment]
The issue is not with the size of the message rather the content. In this json message, there is "null" value which is causing the issue. This is fixed with following PR
https://github.com/ballerinalang/ballerina/pull/2066

**Comment by chanakaudaya** [comment]
Fixed with #2066
